dashboard/src/app/workspaces/create-workspace/project-source-selector/add-import-project/add-import-project.html

```diff
@@ -38,10 +43,12 @@
   <div class="add-import-project-button"
        flex layout="row" layout-align="end center">
     <che-button-cancel-flat che-button-title="Cancel"
+                            id="cancel-Button"
```

Comment:
I think you should always use the same pattern for ids

here is a mix with an uppercase with a `-`character
it should either be cancelButton or cancel-button

maybe it would be easier to stick for all ids to either camelCase or kebab-case

Reply:
Good catch!
Due to the code, _camelCase_ is the most used pattern in the Che. Therefore, "cancelButton" seems more preferable here.

Reply:
Id names were renamed to use pattern with '-' character because dashboard team use it.

Reply:
I see.